src/SFA.DAS.ContentApi.Database/Tables/ClientScope.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[ClientScope]
```

Comment:
Think we should rename to ClientContent just so it is obvious this is a joining table

Reply:
It's called ApplicationContent now